[2022-05-13 15:25:58,665] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: my_dag_v0.0.1.my_task_2 2022-05-13T15:25:46.894949+00:00 [queued]>
[2022-05-13 15:25:58,675] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: my_dag_v0.0.1.my_task_2 2022-05-13T15:25:46.894949+00:00 [queued]>
[2022-05-13 15:25:58,676] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-05-13 15:25:58,676] {taskinstance.py:1068} INFO - Starting attempt 1 of 4
[2022-05-13 15:25:58,676] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-05-13 15:25:58,682] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): my_task_2> on 2022-05-13T15:25:46.894949+00:00
[2022-05-13 15:25:58,691] {standard_task_runner.py:52} INFO - Started process 271 to run task
[2022-05-13 15:25:58,698] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'my_dag_v0.0.1', 'my_task_2', '2022-05-13T15:25:46.894949+00:00', '--job-id', '6', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag_v_0_1_0_0.py', '--cfg-path', '/tmp/tmp3q2p89mg', '--error-file', '/tmp/tmpatobp082']
[2022-05-13 15:25:58,703] {standard_task_runner.py:77} INFO - Job 6: Subtask my_task_2
[2022-05-13 15:25:58,760] {logging_mixin.py:104} INFO - Running <TaskInstance: my_dag_v0.0.1.my_task_2 2022-05-13T15:25:46.894949+00:00 [running]> on host bacb497a28fb
[2022-05-13 15:25:58,832] {taskinstance.py:1282} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=msbeni@gmail.com
AIRFLOW_CTX_DAG_OWNER=mohasa
AIRFLOW_CTX_DAG_ID=my_dag_v0.0.1
AIRFLOW_CTX_TASK_ID=my_task_2
AIRFLOW_CTX_EXECUTION_DATE=2022-05-13T15:25:46.894949+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-13T15:25:46.894949+00:00
[2022-05-13 15:25:58,833] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2022-05-13 15:25:58,834] {subprocess.py:63} INFO - Running command: ['bash', '-c', "echo '1' && exit 0"]
[2022-05-13 15:25:58,843] {subprocess.py:75} INFO - Output:
[2022-05-13 15:25:58,846] {subprocess.py:79} INFO - 1
[2022-05-13 15:25:58,847] {subprocess.py:83} INFO - Command exited with return code 0
[2022-05-13 15:25:58,875] {taskinstance.py:1191} INFO - Marking task as SUCCESS. dag_id=my_dag_v0.0.1, task_id=my_task_2, execution_date=20220513T152546, start_date=20220513T152558, end_date=20220513T152558
[2022-05-13 15:25:58,912] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-05-13 15:25:58,949] {local_task_job.py:151} INFO - Task exited with return code 0
